### ISSUE TITLE
连续两个相同的字符会漏掉一个

### ISSUE THREAD

**Issue by chyun** [Belval/CRNN#23]
连续两个相同的字符会漏掉一个:
例如: aaby识别为aby, 887y识别为87y

**Comment by Belval** [comment]
Change:
https://github.com/Belval/CRNN/blob/063349508bd95e8dfb6b2e1135a7f8d8dd1fbc39/CRNN/crnn.py#L167

To:
`decoded, log_prob = tf.nn.ctc_beam_search_decoder(logits, seq_len, merge_repeated=False)`

And the problem should be fixed.

**Comment by chyun** [comment]
done, thanks!
